CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 ## [0.5] - Unreleased
 
+### Added
```

Comment:
Probably not important, but I don't think the version number in `Cargo.toml`s match up with version number.